internal/command/sandbox/create.go

```diff
@@ -51,34 +52,42 @@ func create(cfg *config.SandboxCreate, out io.Writer) error {
 	}
 	resp := result.Payload
 
-	fmt.Fprintf(out, "Created sandbox %q (sandbox id: %s) in cluster %q.\n\n",
+	fmt.Fprintf(log, "Created sandbox %q (sandbox id: %s) in cluster %q.\n\n",
 		req.Name, resp.SandboxID, *req.Cluster)
 
 	// Print warnings, if any.
 	for _, msg := range resp.Warnings {
-		fmt.Fprintf(out, "WARNING: %s\n\n", msg)
+		fmt.Fprintf(log, "WARNING: %s\n\n", msg)
 	}
 
 	if cfg.Wait {
 		// Wait for the sandbox to be ready.
-		if err := waitForReady(cfg, out, resp.SandboxID); err != nil {
-			fmt.Fprintf(out, "\nThe sandbox was created, but it may not be ready yet. To check status, run:\n\n")
-			fmt.Fprintf(out, "  signadot sandbox get-status %v\n\n", req.Name)
+		if err := waitForReady(cfg, log, resp.SandboxID); err != nil {
+			fmt.Fprintf(log, "\nThe sandbox was created, but it may not be ready yet. To check status, run:\n\n")
+			fmt.Fprintf(log, "  signadot sandbox get-status %v\n\n", req.Name)
 			return err
 		}
 	}
 
-	// Print info on how to access the sandbox.
-	sbURL := cfg.SandboxDashboardURL(resp.SandboxID)
-	fmt.Fprintf(out, "\nDashboard page: %v\n\n", sbURL)
+	switch cfg.OutputFormat {
+	case config.OutputFormatDefault:
```

Comment:
What value does `cfg.OutputFormat` get when:
a. When the flag is provided with just no value. Empty string?
b. When the `--output` flag is not provided. nil / empty?

Just trying to make sure it doesn't land in the default case if `nil` is a possibility causing it to error out.

I took a look at [this](https://github.com/signadot/cli/blob/424bbad106d97e684ab5625470ead9e5e0b784b4/internal/config/root.go#L29) and [this](https://github.com/signadot/cli/blob/4cbf27dc076dd4f3466fd5379a662011b15285a4/internal/config/flag_types.go#L18). But I couldn't be certain.

Reply:
To add to that, if it defaults to an empty string in absence of the `--output` flag, should we write to log in that case?